tests/views/test_ensemble_selector.py

```diff
@@ -12,44 +12,53 @@ def test_ensemble_refresh(
 ):
     plugin = setup_plugin(dash_duo, __name__, ParameterComparison)
 
-    ensemble_selector_id = plugin.uuid("ensemble-multi-selector")
-    initial_cases = get_options(dash_duo, "#" + ensemble_selector_id)
+    ensemble_selector_id = f"#{plugin.uuid('ensemble-multi-selector')}"
+    expected_initial_ensembles = [
+        "default",
+        "default_smoother_update",
+        "default3",
+        "nr_42",
+    ]
+    dash_duo.wait_for_text_to_equal(
+        ensemble_selector_id, "\n".join(expected_initial_ensembles)
+    )
 
     # Select first ensemble
     first_ensemble_name = select_ensemble(dash_duo, plugin)
 
     # Select a parameter
-    param_name = select_first(
+    first_parameter = select_first(
         dash_duo, "#" + plugin.uuid("parameter-selector-multi-param")
     )
 
-    dash_duo.wait_for_contains_text(
-        "#" + plugin.uuid("parameter-deactivator-param"), f"×{param_name}", timeout=4
+    selected_parameters_finder = (
+        f"#{plugin.uuid('parameter-deactivator-param')} " ".Select-value-label"
```

Comment:
do we actually have names ( css classes ) with a capital first letter or is this mispelled?

Reply:
i think that's not us as in webviz-ert, it's the `Dropdown` component from `dash-core-components`, but yeah, we do:

![css-class-names](https://user-images.githubusercontent.com/15932677/185348433-28247380-7209-4c1f-913d-0046a01047ce.png)